reviews/layout/helpers.py

```diff
@@ -19,15 +19,24 @@ def render_pull_request_table(
     """Renders a list of pull requests as a table"""
     table = Table(show_header=True, header_style="bold white")
     table.add_column("#", style="dim", width=5)
-    table.add_column(title, width=60)
+    table.add_column(title, width=55)
     table.add_column("Labels", width=40)
     table.add_column("Activity", width=15)
-    table.add_column("Status", width=10)
+    table.add_column("Approved", width=15)
+    table.add_column("Ready To Release", width=10)
 
     pull_requests = sorted(pull_requests, key=lambda x: x.updated_at, reverse=True)
 
     for pr in pull_requests:
-        approved = "[green]Approved" if pr.approved else ""
+        approved = (
```

Comment:
It would be good to refactor this snippet to improve readability:

```python
        approved = ""
        if pr.approved == "APPROVED":
            approved = "[green]Approved"
        elif pr.approved == "CHANGES_REQUESTED"
            approved = "[yellow]Changes Requested"
```